### PR TITLE
[Dungeon] The Stockade

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588336883299094900.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 116, `maxdmg` = 149, `DamageModifier` = 1.02 WHERE `entry` = 1720;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1696;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1666;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1663;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1716;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 1717;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336883299094900.sql:23]: fix newline
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 1717;

```

File: data/sql/updates/pending_db_world/rev_1588336883299094900.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1588336883299094900');
+
+/*
+ * Dungeon: Stormwind Stockade
+ * Update by Knindza | <www.azerothcore.org>
+*/
+
+/* REGULAR */ 
+UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` = 1707;
+UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` = 1706;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336883299094900.sql:10]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` IN (1707, 1706);
```

File: data/sql/updates/pending_db_world/rev_1588336883299094900.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 84, `DamageModifier` = 1.03 WHERE `entry` = 1708;
+UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 84, `DamageModifier` = 1.03 WHERE `entry` = 1711;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336883299094900.sql:12]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 84, `DamageModifier` = 1.03 WHERE `entry` IN (1708, 1711);
```

File: data/sql/updates/pending_db_world/rev_1588336883299094900.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+UPDATE `creature_template` SET `mindmg` = 82, `maxdmg` = 109, `DamageModifier` = 1.03 WHERE `entry` = 1715;
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 116, `maxdmg` = 149, `DamageModifier` = 1.02 WHERE `entry` = 1720;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1696;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1666;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1663;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1716;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336883299094900.sql:22]:
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` IN (1666, 1663, 1716);
```

**Comment by Geekena** [comment]
damage is good but agrro is problem
when moving they agrro me trough walls and there is no save path 
agrro make ogree boss very very hard

**Comment by Knindzagxg** [comment]
Aggro Radius can be changed in worldserver.conf
```
#    Rate.Creature.Aggro
#        Description: Aggro radius percentage.
#        Default:     1   - (Enabled, 100%)
#                     1.5 - (Enabled, 150%)
#                     0   - (Disabled, 0%)

Rate.Creature.Aggro = 0.5
```

Personally i use 0.5, cause it makes game much more realistic.
Default is 1, and in near future i will do more tests to be sure what % we could use as default.

File: data/sql/updates/pending_db_world/rev_1588336883299094900.sql [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+UPDATE `creature_template` SET `mindmg` = 60, `maxdmg` = 84, `DamageModifier` = 1.03 WHERE `entry` IN (1708, 1711);
+UPDATE `creature_template` SET `mindmg` = 82, `maxdmg` = 109, `DamageModifier` = 1.03 WHERE `entry` = 1715;
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 116, `maxdmg` = 149, `DamageModifier` = 1.02 WHERE `entry` = 1720;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1696;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` IN (1666, 1663, 1716);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 1717;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336883299094900.sql:19]: change to `type_flags`=`type_flags`|4

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1588336883299094900.sql:19]: Please change to what I said above

File: data/sql/updates/pending_db_world/rev_1588336883299094900.sql [hunk 6]
```diff
@@ -14,6 +14,6 @@ UPDATE `creature_template` SET `mindmg` = 82, `maxdmg` = 109, `DamageModifier` =
 UPDATE `creature_template` SET `rank` = 2, `mindmg` = 116, `maxdmg` = 149, `DamageModifier` = 1.02 WHERE `entry` = 1720;
 
 /* BOSS */
-UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1696;
-UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` IN (1666, 1663, 1716);
-UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 1717;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 106, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 1696;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` IN (1666, 1663, 1716);
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 1717;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336883299094900.sql:19]: now some \` were removed, add them around all type_flags where it is missing

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1588336883299094900.sql]: Done xD

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**